api/src/main/java/org/openmrs/ServiceOrder.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+/**
+ * Provides properties for several order types like TestOrder , ReferralOrder
+ * and others to accommodate referrals depending on the openmrs implementation
```

Comment:
`to accommodate referrals` sounds like you are using service order for only referalls.